### PR TITLE
Feature/helmet

### PR DESCRIPTION
**Pull request by palmfjord**
⛑

### CODE REVIEW

File: src/serverEntry.tsx [hunk 1]
```diff
@@ -43,6 +46,22 @@ const server = createKoaServer({
...
+    extendTypes: { json: ['application/csp-report'] },
+  }),
+)
+
+if (process.env.USE_HELMET === 'true') {
+  appLogger.info('Using helmet and strict CSP ⛑')
+  server.app.use(helmet)
+  server.router.use(helmet)
+} else if (process.env.NODE_ENV !== 'development') {
+  appLogger.warn(
```
Comment (hedvigoscar) [inline src/serverEntry.tsx]: I would even consider bailing at this point. Going without in production would not be great as the log statement says!

File: src/types/koa.d.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import 'koa'
+
+declare module 'koa' {
```
Comment (hedvigoscar) [inline src/types/koa.d.ts]: 🙏

File: src/types/koa-bodyparser.d.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+declare module 'koa-bodyparser' {
```
Comment (hedvigoscar) [inline src/types/koa-bodyparser.d.ts]: :pray:

File: src/serverEntry.tsx (hunk 1 above)
Comment (palmfjord) [reply src/serverEntry.tsx]: I don't agree. I.e. if we're putting up a PR instance or want to try something locally it's not obvious we care about CSPs.

**Pull request merged**